x-pack/plugins/apm/public/components/shared/charts/most_used_chart/get_lens_attributes.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CountIndexPatternColumn,
+  TermsIndexPatternColumn,
+  PersistedIndexPatternLayer,
+  PieVisualizationState,
+  TypedLensByValueInput,
+} from '@kbn/lens-plugin/public';
+import type { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+import { APM_STATIC_DATA_VIEW_ID } from '../../../../../common/data_view_constants';
+import { MostUsedMetric } from '.';
+
+export function getLensAttributes({
```

Comment:
It might be better to be moved to the service overview folder as it's only used there for now.

Reply:
make sense, will move it